src/docs/truffle/index.md

```diff
@@ -12,12 +12,14 @@ layout: docs.hbs
 A world class development environment, testing framework and asset pipeline for blockchains using the Ethereum Virtual Machine (EVM), aiming to make life as a developer easier. With Truffle, you get:
 
 * Built-in smart contract compilation, linking, deployment and binary management.
+* Advanced debugging with breakpoints, variable analysis, and step functionality.
+* Deployments and transactions through MetaMask to protect your mnemonic.
+* External script runner that executes scripts within a Truffle environment.
+* Interactive console for direct contract communication.
 * Automated contract testing for rapid development.
 * Scriptable, extensible deployment & migrations framework.
 * Network management for deploying to any number of public & private networks.
-* Package management with EthPM & NPM, using the [ERC190](https://github.com/ethereum/EIPs/issues/190) standard.
-* Interactive console for direct contract communication.
```

Comment:
You sure this one should be removed?

Reply:
I moved the console one up in priority since people seem to forget that functionality exists

ETHPM is deprecated